cli/build.rs

```diff
@@ -366,7 +366,7 @@ fn create_cli_snapshot(snapshot_path: PathBuf) {
     // FIXME(bartlomieju): information about which extensions were
     // already snapshotted is not preserved in the snapshot. This should be
     // fixed, so we can reliably depend on that information.
-    // .dependencies(vec!["runtime"])
+    .dependencies(vec!["deno_websocket"])
```

Comment:
Why changed to `deno_websocket` instead of runtime?

Reply:
runtime isnt an ext

Reply:
It seems it is: https://github.com/denoland/deno/blob/0aeb8bc7591ec32e199b98f4532d6a962c6da6cb/runtime/build.rs#L169

Reply:
oh you are right; overlooked as its defined in the snapshot. will revert it to runtime then